src/main/webapp/app/overview/course-dashboard/course-exercise-performance/course-exercise-performance.component.ts

```diff
@@ -0,0 +1,104 @@
+import { Component, Input, OnChanges, OnInit } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { Color, ScaleType } from '@swimlane/ngx-charts';
+import { GraphColors } from 'app/entities/statistics.model';
+import { NgxChartsMultiSeriesDataEntry } from 'app/shared/chart/ngx-charts-datatypes';
+import { round } from 'app/shared/util/utils';
+
+export interface ExercisePerformance {
+    exerciseId: number;
+    title: string;
+    shortName?: string;
+    score?: number;
+    averageScore?: number;
+}
+
+const YOUR_GRAPH_COLOR = GraphColors.BLUE;
+const AVERAGE_GRAPH_COLOR = GraphColors.YELLOW;
+
+@Component({
+    selector: 'jhi-course-exercise-performance',
+    templateUrl: './course-exercise-performance.component.html',
+    styleUrls: ['./course-exercise-performance.component.scss'],
+})
+export class CourseExercisePerformanceComponent implements OnInit, OnChanges {
+    @Input() exercisePerformance: ExercisePerformance[] = [];
```

Comment:
This component seems copy pasted from `jhi-course-exercise-performance`. I think we should have a shared component for the table. There are really not many differences.